exp-player/addon/components/exp-player.js

```diff
@@ -142,6 +143,7 @@ able to continue the study.
         next() {
             console.log('next');
             var frameIndex = this.get('frameIndex');
+            this.sendAction('updateFrameIndex', frameIndex);
```

Comment:
This action gets bubbled up to the `participate.survey.index` controller, which handles the action: https://github.com/samanehsan/isp/blob/6a470e32d4101b6f3058b6123e8ee835d9510c8e/app/controllers/participate/survey/index.js#L19